src/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-family: "Avenir", "Arial", sans-serif;
+}
+
+.search__bar {
+  position: relative;
+  display: block;
+  margin-top: 20px;
+}
+
+.search__icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.search__icon--small {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+}
+
+.search__field {
+  font-family: inherit;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  width: 100%;
+  height: 70px;
+  padding: 25px 62px;
+  font-size: 16px;
+}
+
+.search__field--small {
+  height: 42px;
+  padding: 12px 33px;
+  font-size: 14px;
+}
+
+.search__field::placeholder {
+  color: #3d4e61;
```

Comment:
Set the font wight to avoid it
![image](https://user-images.githubusercontent.com/98764641/199246549-a88ea348-54a4-40da-860d-41bc82efa05c.png)

Reply:
Isn't `color ` here redundant?

Reply:
Not need)